test/gpu/native/studies/chop/chplGPU.skipif

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
-# Skip if not doing performance testing
-if [ -z "$CHPL_TEST_PERF" ]; then
+if [ "$CHPL_GPU" = "cpu" ]; then
```

Comment:
What was the issue with cpu-as-device?

Reply:
My own lack of faith in cpu-as-device mode it looks like. Just tried it and it looks like this mode is just fine. I've taken that bit out.